### ISSUE TITLE
Correct the website uploaded date on the registry page

### ISSUE THREAD

**Issue by SlnPons** [massalabs/thyra#268]
Thyra on Testnet 16 displayed a date in 1970 which is a problem that needs to be solved before we launch Thyra to all.

The expected behaviour is display: 'date of the day'

EDIT :  

We need to use args class in JS frontend, we cannot properly access to massa-web3 in vanilla js with the current export of args 

Decided with Solene :  

Hide Date column

Make available the serialized data in the frontend to have a fast implementation when args available

**Comment by SlnPons** [comment]
As discussed together with @SebastienValla, we'll do the following: 
1- temporary solution to unblock thyra for all launch to the community: remove the timestamp information from the displayed table (ie: remove this column from the UI) 
2- long terme solution to fix this properly when the args could be used with the framework then will fix this by deserialising the string from the back-end to the front-end. @SebastienValla this is not well explained....sorry could you rephrase this to make it clear please? 
Also, can you create an issue to address the long term solution in Thyra repo please? 
Thanks a lot

**Comment by SebastienValla** [comment]
Done !

**Comment by SlnPons** [comment]
@SebastienValla can you share the ticket you created about the long term vision?
